source/administration/migrating.rst

```diff
@@ -48,13 +48,13 @@ This feature can be accessed through the `Mattermost Web App <https://docs.matte
    If you don't use Docker, there are `step-by-step instructions <https://docs.mattermost.com/install/docker-local-machine.html>`_ to install Mattermost in preview mode in less than 5 minutes.
 
 Supported Features
-++++++++++++++++++
+~~~~~~~~~~~~~~~~~~
 
 The following key features are supported when importing from Slack:
 
 * User accounts with an email address set
 
-* Public channels and the text messages posted in them
+* Public channels and the text messages posted in them, with formatting
```

Comment:
This PR https://github.com/mattermost/platform/pull/4914 translates their non-standard markdown to our formatting. It would still be good to test though.